content/appointments.md

```diff
@@ -1,49 +1,49 @@
 ---
-description: Free and impartial Pension Wise guidance appointments can help you understand your pension options. Find out how they work, who’s eligible and what you need to do to prepare.
+description: Book a free and impartial Pension Wise guidance appointment to help you understand your pension options.
 ---
 
-#Pension Wise guidance appointments
```

Comment:
We need a primary heading (H1) otherwise the page makes no sense if you navigate to it directly.

![screen shot 2015-04-20 at 08 58 47](https://cloud.githubusercontent.com/assets/2715/7225936/86a8b684-e73b-11e4-85aa-3387dbf9076e.png)